lib/ButtonNew/stories/ButtonIcon/ButtonIconDark.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { ButtonIconDark } from '../../ButtonIcon/ButtonIconDark';
+import StoryItem from '../../../../stories/styleguide/StoryItem';
+import { ButtonStoryItem } from '../ButtonStoryItem';
+import { Counter } from '../../../Counter';
+
+storiesOf('Components', module).add('ButtonIconDark', () => (
```

Comment:
🟢  What do you think to have all the `ButtonIcon`s in one story?

Reply:
Yeah this could be nice!